src/main/webapp/app/guided-tour/guided-tour.component.scss

```diff
@@ -157,6 +157,11 @@ jhi-guided-tour {
                     @media (max-height: 600px) {
                         max-height: 75vh;
                     }
+
+                    iframe {
```

Comment:
How does this affect other screen sizes? e.g. if someone has 16/10 or 4/3 (lol)

Reply:
It worked on my laptop (16/10) as well. This is only the video itself and not the whole site